everest

```diff
@@ -745,6 +719,17 @@ OCAML
     magenta "Note: you *may* want to export HACL_HOME=${xpwd}/hacl-star"
 }
 
+get_vale ()
+{
+  mkdir vale
```

Comment:
what happens if the vale directory already exists?

Reply:
Sigh.  I was about to reply that mkdir is supposed to return with no error if the directory already exists, based on the documentation I saw for mkdir.  But I can see that this isn't what actually happens under Cygwin.  I'll add the necessary check.